ci/rake_release.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Please:

```sh
set -e
```

In all bash scripts used in CI. This ensures that when a nonzero exit code is detected, the script stops and exits with that same code. (Otherwise, we'll see test failures in the logs but the actual CI job won't fail.)

Reply:
Done! This revealed some errors with the mysql service which I also fixed.